app/src/js/widgets/fields/fieldVideo.js

```diff
@@ -107,11 +107,13 @@
          */
         _update: function () {
             var self = this,
-                url = 'https://api.embed.ly/1/oembed',
+                url = bolt.conf('paths.async') + 'oembed/request',
```

Comment:
I've been trying to get away from string concatenation with paths. Can you assign url generator result to a data attribute in twig and fetch that here?